### PR TITLE
EQL: [Tests] Cleanup and optimise code

### PR DESCRIPTION
**Pull request by matriv**
Cleanup unused code and minor code optimisations.

### CODE REVIEW

File: x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/search/OrdinalTests.java [hunk 1]
```diff
@@ -12,23 +12,26 @@
 @SuppressWarnings("unchecked")
 public class OrdinalTests extends ESTestCase {
 
+    @SuppressWarnings("rawtypes")
```
Comment (costin) [inline x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/search/OrdinalTests.java:15]: I don't find any value in adding the annotations at method level - it happens for all methods hence why it's defined at the class level.

**Review by costin — APPROVED** [review]
LGTM

File: x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/search/OrdinalTests.java (hunk 1 above)
Comment (matriv) [reply x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/search/OrdinalTests.java:15]: Agreed, it's only one method that doesn't use it, so I'll move it to the class instead.

**Review by bpintea — APPROVED** [review]
LGTM

**Pull request merged**